code/modules/components/ai/human/human_target_finder.dm

```diff
@@ -18,4 +18,14 @@
 			var/mob/living/carbon/human/H = M
 			if(H.species && H.species.name in B.enemy_species)
 				o += M
-	return o
\ No newline at end of file
+	return o
+
+/datum/component/ai/target_finder/human/RecieveSignal(var/message_type, var/list/args)
+	..()
+	if(message_type == COMSIG_ATTACKEDBY) //YOU HAVE JUST MADE AN ENEMY FOR LIFE
+		var/assailant = args["assailant"]
+		var/damage_done = args["damage"]
+		if(damage_done > 15) //Intent to kill!
+			B.friends &= assailant
```

Comment:
@DamianX Does this work? I'm not familiar with these operators regarding lists.

Reply:
You originally had `B.friends.Remove(assailant)`, which was fine.
The equivalent would be `-=` but I did not request this change. I just said the `Find()` was not needed.
So no, this does not work.